baselines/ppo1/pposgd_simple.py

```diff
@@ -133,29 +143,34 @@ def learn(env, policy_fn, *,
         for (oldv, newv) in zipsame(oldpi.get_variables(), pi.get_variables())])
     compute_losses = U.function([ob, ac, atarg, ret, lrmult], losses)
 
-    U.initialize()
+    initialize_uninitialized(tf.get_default_session())
```

Comment:
This line is very important to make a pre-trained network works properly. The problem of `U.initialize()` - or even the similar method in tensorflow - is because it initializes the whole graph, making all the weights random again (screwing pre-training stuff). However, if I simply remove it, we do not initialize the exploration parameters, so I created this method that solves this problem.

Reply:
Cool! Thanks for the explanation.